scripts/deploy/actions/hugepage-enabler-daemonset.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
```

Comment:
Shouldn't this be part of azure node setup, instead of being in mayastor deployment scripts?